### DISCUSSION TITLE
The language of the error message is not fixed, sometimes returned in English and sometimes in Japanese

### DISCUSSION THREAD

**Discussion by DavidWeng916** [pixiv/vroid-sdk-developers#85] [Q&A]
Hi, I would like to ask a question, when I use **OAuthClient.Login()** API, when there is an error, I find that the error message is sometimes returned in English and sometimes in Japanese.
Is there any way to return only English? VRoid SDK version 0.1.4 and 0.2.0 are the same.

It will look like this:
`{"error":"invalid_grant","error_description":"The provided authorization grant is invalid, expired, revoked, does not match the redirection URI used in the authorization request, or was issued to another client."}`
or
`{"error":"invalid_grant","error_description":"指定された認可グラントは無効か、有効期限切れか、リダイレクトURIが異なるか、もしくは別のクライアントに適用されています。"}`

**Comment by nyamadan** [comment]
Hi, the language of error messages depends on [VRoidHub user language settings](https://hub.vroid.com/settings).
There is no way to fix the language of error messages.

We will consider implementing a way to specify the language of error messages.

**Reply by DavidWeng916** [reply]
I see that the language I set now is English, but I also receive error messages in Japanese, it feels a bit random, sometimes I receive English and sometimes Japanese

**Reply by nyamadan** [reply]
We will investigate.

**Reply by DavidWeng916** [reply]
Thank you a lot 🙏

**Reply by DavidWeng916** [reply]
Hello, about this issue, are there any follow-up results of the survey?

**Reply by nyamadan** [reply]
> I see that the language I set now is English, but I also receive error messages in Japanese, it feels a bit random, sometimes I receive English and sometimes Japanese

We created an issue ticket in our task system and assigned an engineer.
The problem has been reproduced and is being investigated.

We will be able to specify an error language for the login API.
